app/services/attribute_asserter.rb

```diff
@@ -59,10 +60,18 @@ def verified_at_getter_function
     ->(principal) { principal.active_profile.verified_at.iso8601 }
   end
 
+  def pick_getter_function(attr, use_ascii)
+    use_ascii ? attribute_getter_function_ascii(attr) : attribute_getter_function(attr)
```

Comment:
Looks like code climate does not like this....maybe we could move the ternary up?

```ruby
getter = use_ascii ? attribute_getter_function_ascii(attr) : attribute_getter_function(attr)
attrs[attr] = { getter: getter }
```